developer-community/tech-talks.md

```diff
@@ -0,0 +1,240 @@
+# The full story
+
+# Speakers
+## Contact
+
+Ideally, we should start contacting speakers __2.5 months__ prior to the event.
```

Comment:
Will put 3 months and I think ppl know that the sooner you start the better chances you have. I just don't want to discourage anyone with less than that to start organising something because with Infra we did everything (and it was the first) in 2 months, ML we did in 2 as well. Only front we started very early and still didn't get better speakers than the others so I don't think with just 3 talks we can generalise too much anything yet :)